### DISCUSSION TITLE
Unable to find the .mpb and SMLets .dll

### DISCUSSION THREAD

**Discussion by bryanmccuen** [AdhocAdam/smletsexchangeconnector#415] [Q&A] [answered]
Hello,

I think I might just be blind or I did not read the wiki correctly because I can't seem to find the .mpb file or the SMlet .dll in the management pack folder. Where can I find these? Thanks in advance.

**Comment by AdhocAdam** [comment, marked as answer]
I think the difference you're noticing here is between the Github repo itself vs. the releases page.

- The repo contains all of source code used to create and assemble the MPB, DLL, etc. This is how others can contribute, review, and gain knowledge on the inner workings of the project.
- The Releases page is essentially the output. It's the finalized ready to deploy bits you'll need. So just head over the [Release page](https://github.com/AdhocAdam/smletsexchangeconnector/releases) and download the zip file for the version you want to download. It's in the zip you'll find the MPB and DLL.

Hope that helps out. If not, let me know!

**Reply by bryanmccuen** [reply]
I guess it would help if I was in the right place! Thanks so much!

**Reply by AdhocAdam** [reply]
No worries. Glad to help 😄
